README.md

```diff
@@ -1,6 +1,22 @@
+<div align="center">
+  <img src="./assets/icons/logo.svg" width="320">
+</div>
+
 # Kubeflow Website
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/80644d22-0685-44d0-83bd-187d55464321/deploy-status)](https://app.netlify.com/sites/competent-brattain-de2d6d/deploys)
+<h4 align="center">
+
+  [![Netlify Status](https://api.netlify.com/api/v1/badges/80644d22-0685-44d0-83bd-187d55464321/deploy-status)](https://app.netlify.com/sites/competent-brattain-de2d6d/deploys)
+  <a href="https://landscape.cncf.io/?item=orchestration-management--scheduling-orchestration--kubeflow">
+    <img src="https://img.shields.io/badge/CNCF%20Landscape-5699C6?logo=cncf&style=social" alt="KubeFlow CNCF Landscape" />
+  </a>
+
+  [![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](./docs/about/community/#kubeflow-slack-channels)
+  [![LinkedIn](https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/company/kubeflow)
+  [![YouTube](https://img.shields.io/badge/YouTube-%23FF0000.svg?style=for-the-badge&logo=YouTube&logoColor=white)](https://www.youtube.com/kubeflow)
+  [![X](https://img.shields.io/badge/X-%23000000.svg?style=for-the-badge&logo=X&logoColor=white)](https://x.com/kubeflow/)
+
```

Comment:
Please also add this:
https://bsky.app/profile/kubefloworg.bsky.social

Reply:
Added bluesky